src/web/controllers/admin/prize_admin_controller.py

```diff
@@ -203,7 +207,12 @@ async def htmx_admin_prizes_tab(
         event_uniq_id: str,
         tournament_id: int | None,
         prize_group_id: int | None,
+        admin_prizes_show_details: bool | None,
```

Comment:
```suggestion
        show_details: bool | None,
```
it's a query param, mentioning `admin` in it feels redundant